Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39624&attempt_id=0342d4c4-c880-47a9-b125-1bed464b92e1)

Attempted: kcov

<details><summary>Partial log (click to expand)</summary><p>

```
                                                ^~~~~~~~~~~
                                                ADJ_STATUS
make[2]: *** [src/CMakeFiles/kcov.dir/build.make:708: src/CMakeFiles/kcov.dir/engines/ptrace.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:244: src/CMakeFiles/kcov-system-daemon.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:207: src/CMakeFiles/kcov.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
builder for '/nix/store/n5p0326sqza7r9y9izwcav7k2i7syvg4-kcov-35.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/n5p0326sqza7r9y9izwcav7k2i7syvg4-kcov-35.drv' failed
```
</p></details>

--- a/pkgs/development/tools/analysis/kcov/aarch64_nt_prstatus.patch
+++ b/pkgs/development/tools/analysis/kcov/aarch64_nt_prstatus.patch
@@ -1,0 +1,12 @@
+diff --git a/src/engines/ptrace.cc b/src/engines/ptrace.cc
+index 59b615f..e02cddf 100644
+--- a/src/engines/ptrace.cc
++++ b/src/engines/ptrace.cc
+@@ -21,6 +21,7 @@
+ 
+ #if defined(__aarch64__)
+ #  include <sys/uio.h>
++#  include <elf.h>
+ #endif
+ 
+ #include <map>

--- a/pkgs/development/tools/analysis/kcov/default.nix
+++ b/pkgs/development/tools/analysis/kcov/default.nix
@@ -16,6 +16,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ zlib curl elfutils python libiberty libopcodes ];
 
+  patches = [ ./aarch64_nt_prstatus.patch ];
+
   enableParallelBuilding = true;
 
   meta = with stdenv.lib; {

